.github/workflows/general.yml

```diff
@@ -57,6 +57,9 @@ jobs:
       - name: Install CAD Suite (includes yosys)
         run: tool/gh_actions/install_opencadsuite.sh
 
+      - name: Install SystemVerilog to Verilog converter
+        run: tool/gh_actions/install_sv2v.sh
```

Comment:
do you want to add this to `tool/gh_codespaces/run_setup.sh` as well so the container has it too?